src/openzaak/config/setup_configuration/steps/selectielijst.py

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+from zgw_consumers.models import Service
+
+from openzaak.config.setup_configuration.models import SelectielijstAPIConfig
+from openzaak.selectielijst.models import ReferentieLijstConfig
+
+
+def get_service(slug: str) -> Service:
+    """
+    Try to find a Service and re-raise DoesNotExist with the identifier
+    to make debugging easier
+    """
+    try:
+        return Service.objects.get(slug=slug)
+    except Service.DoesNotExist as e:
+        raise Service.DoesNotExist(f"{str(e)} (identifier = {slug})")
```

Comment:
let's reraise this as `django_setup_configuration.exceptions.ConfigurationRunFailed`